src/containers/Masthead/drawer/DefaultMenu.tsx

```diff
@@ -99,37 +84,33 @@ const DefaultMenu = ({
   useArrowNavigation(!type, {
     initialFocused: `header-${dynamicId ?? type ?? previousType ?? "programme"}`,
     onRightKeyPressed: onRightClick,
+    multilevel: true,
```

Comment:
Er det en grunn til at du har lagt til denne?

Reply:
For at navigasjonen skal fungere med de nye span headerene som vi har på første nivået her.